med_io/parser_tfrec.py

```diff
@@ -11,11 +11,15 @@ def parser(example_serialized):
     # features['image'] must be tf.float and features['image_shape'] must be tf.int32!
     features_map = {
         'image': tf.io.FixedLenFeature([], tf.string),
-        'image_shape': tf.io.FixedLenFeature([], tf.string)
+        'image_shape': tf.io.FixedLenFeature([], tf.string),
+        'validation_for_cancer': tf.io.FixedLenFeature([], tf.int64)
```

Comment:
no backward comp.!  Do not overwrite previous behaviour -> move to separate function

Reply:
Inside the parser_tfrec.py file, I added a new function "parser_tfrec_melanom" which is then used in the pipeline for MELANOM dataset. Original parser function is again the default one.